deployment/roles/kafka/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+server_num: "1"
+kafka_instance:
+  zookeeper_master_ip: "{{ cluster_host }}"
+  zookeeper_master_port: 2181
```

Comment:
Here is the default value for this variable so I can't substitute default value for some other value. If new value of variable will be passed to the role it will supress this one - http://docs.ansible.com/playbooks_roles.html